doc/man/s3fs.1

```diff
@@ -215,6 +215,12 @@ sets MB to ensure disk free space. This option means the threshold of free space
 s3fs makes file for downloading, uploading and caching files.
 If the disk free space is smaller than this value, s3fs do not use diskspace as possible in exchange for the performance.
 .TP
+\fB\-o\fR fake_diskfree (default 0)
```

Comment:
We shouldn't add these kinds of test-only options to the help or man page.  It will just confuse users.

Reply:
OK, It will be removed soon.